app-frontend/src/app/components/projects/projectLayerNavbar/index.html

```diff
@@ -0,0 +1,43 @@
+<div class="navbar light-navbar">
```

Comment:
I'm gonna rename this because it's confusable with @aaronxsu 's projectLayersNav component